mapstruct-on-ant/build.xml

```diff
@@ -0,0 +1,109 @@
+     http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<project name="mapstruct-on-ant" default="build" basedir=".">
+    <property name="mapstruct.version" value="1.0.0.Final" />
+
+    <property name="src.main.dir" location="src/main/java" />
+    <property name="src.test.dir" location="src/test/java" />
+
+    <property name="build.dir" location="target" />
+    <property name="classes.dir" location="${build.dir}/classes" />
+    <property name="generatedsources.dir" location="${build.dir}/generated-sources" />
+    <property name="test.classes.dir" location="${build.dir}/test-classes" />
+    <property name="test.report.dir" location="${build.dir}/junit-reports" />
+    <property name="lib.dir" location="${user.home}/.m2/repository" />
+
+    <path id="main.classpath">
+        <fileset dir="${lib.dir}">
+            <include name="org/mapstruct/mapstruct-jdk8/${mapstruct.version}/mapstruct-jdk8-${mapstruct.version}.jar" />
+        </fileset>
+    </path>
+
+    <path id="processor.path">
+        <fileset dir="${lib.dir}">
+            <include name="org/mapstruct/mapstruct-processor/${mapstruct.version}/mapstruct-processor-${mapstruct.version}.jar" />
+        </fileset>
+    </path>
+    <pathconvert property="processorpath" refid="processor.path"/>
+        
+    <path id="test.classpath">
+        <fileset dir="${lib.dir}">
+            <include name="junit/junit/4.12/junit-4.12.jar" />
+            <include name="org/easytesting/fest-assert/1.4/fest-assert-1.4.jar" />
+            <include name="org/easytesting/fest-util/1.1.6/fest-util-1.1.6.jar" />
+            <include name="org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar" />
+            <include name="org/mapstruct/mapstruct-jdk8/${mapstruct.version}/mapstruct-jdk8-${mapstruct.version}.jar" />
+        </fileset>
+        <pathelement location="${classes.dir}" />
+        <pathelement location="${test.classes.dir}" />
+    </path>
+
+    <target name="clean">
+        <delete dir="${build.dir}" />
+    </target>
+
+    <target name="create-directories">
+        <mkdir dir="${build.dir}" />
+        <mkdir dir="${classes.dir}" />
+        <mkdir dir="${generatedsources.dir}" />
+        <mkdir dir="${test.classes.dir}" />
+        <mkdir dir="${test.report.dir}" />
+    </target>
+
+    <target name="compile" depends="clean, create-directories">
+        <javac
+            srcdir="${src.main.dir}"
+            destdir="${classes.dir}"
+            classpathref="main.classpath"
+            includeantruntime="false">
+            <compilerarg line="-processorpath ${processorpath}"/>
+            <compilerarg line="-processor org.mapstruct.ap.MappingProcessor"/>
```

Comment:
How about adding a comment explaining that it would only be needed if there are other processors on the processorpath and only this one would need to be executed? I'd favor that so that it doesn't appear to be more complicated than it has to be... I could do it tomorrow, but for now my day is over 😴